wielder/util/terraform_credential_helper.py

```diff
@@ -65,7 +65,18 @@ def get_aws_mfa_cred_command(role_name):
                         env_cred["AWS_SESSION_TOKEN"] = cred["SessionToken"]
                         env_cred["AWS_SECURITY_TOKEN"] = cred["SessionToken"]
 
-                        break
+                        return env_cred
+
+
+def get_aws_mfa_cred_command(role_name):
+    """
+    Looks for aws MFA session credentials file,
+    extracts env variables necessary for running terraform
+    :param role_name: role name to check against CLI MFA cache.
+    :return: returns env credential shell command or empty string.
+    """
+
+    env_cred = get_aws_mfa_cred(role_name)
 
     as_string_cmd = as_export_cmd(env_cred)
```

Comment:
Rename something like:credential_to_cmd

Reply:
renamed and renamed module to better reflect the level of abstraction and reusability